rhmap-test-driver/app/index.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+const Utils = require("./src/utils");
+const TestRunner = require("./src/test-runner");
+
+const appId = process.argv[2];
+const csvPath = process.argv[3];
+
+if (!appId || !csvPath) {
+    console.log("usage: node index.js <appId> <path/to/devices.csv>");
```

Comment:
The applicationId, the id of the application (kidding). This parameter is required for $fh.